stock_inventory_analytic_warehouse_default/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ACSONE SA/NV
```

Comment:
Minor-comment:
Shouldn't the dates be 2023?

Reply:
@FrancoMaxime Hehe, module was developped at the end of 2021. I've added improvements today